modules/ROOT/pages/getting-started-tutorial.adoc

```diff
@@ -0,0 +1,307 @@
+
+[source,bash]
+----
+[core@localhost]$ rpm-ostree status
+State: idle
+AutomaticUpdates: disabled
+Deployments:
+* ostree://fedora:fedora/x86_64/coreos/stable
+                   Version: 31.20200108.3.0 (2020-01-09T21:51:07Z)
+                    Commit: 113aa27efe1bbcf6324af7423f64ef7deb0acbf21b928faec84bf66a60a5c933
+              GPGSignature: Valid signature by 7D22D5867F2A4236474BF7B850CB390B3C3359C4
+----
+
+*Note:* The ostree version can differ
+
+And check on zincati.service, which communicates with our update server and tells rpm-ostree when to do an update and to what version to update to:
+
+[source,bash]
+----
+[core@localhost]$ systemctl status zincati.service | cat
+● zincati.service - Zincati Update Agent
+   Loaded: loaded (/usr/lib/systemd/system/zincati.service; enabled; vendor preset: enabled)
+   Active: active (running) since Mon 2020-01-20 20:01:11 UTC; 2min 9s ago
+     Docs: https://github.com/coreos/zincati
+ Main PID: 1063 (zincati)
+    Tasks: 2 (limit: 2297)
+   Memory: 11.4M
+   CGroup: /system.slice/zincati.service
+           └─1063 /usr/libexec/zincati agent -v
+
+Jan 20 20:01:11 localhost systemd[1]: Started Zincati Update Agent.
+Jan 20 20:01:11 localhost zincati[1063]: [INFO ] starting update agent (zincati 0.0.6)
+Jan 20 20:01:14 localhost zincati[1063]: [INFO ] Cincinnati service: https://updates.coreos.stg.fedoraproject.org
+Jan 20 20:01:14 localhost zincati[1063]: [INFO ] agent running on node 'ead8e3b8ead045dcb36c04bc7be0acbc', in update group 'default'
+Jan 20 20:01:14 localhost zincati[1063]: [WARN ] initialization complete, auto-updates logic disabled by configuration
+----
+
+*Note:* You can see from the "auto-updates logic disabled by configuration" message that zincati properly picked up the configuration to disable updates that was placed in /etc/zincati/config.d/90-disable-auto-updates.toml.
+
+One other interesting thing to do is view the logs from Ignition in case there is anything interesting there you may want to investigate:
+
+[source,bash]
+----
+[core@localhost]$ journalctl -t ignition | cat
+----
+
+And finally, of course you can use the docker (provided by moby-engine rpm) or podman commands to inspect the current state of containers on the system:
+
+[source,bash]
+----
+[core@localhost]$ podman version
+[core@localhost]$ podman info
+[core@localhost]$ sudo docker info
+[core@localhost]$ sudo docker version
+----
+
+*Note:* You need `sudo` for docker commands. `podman` commands can be run as root or as non-root user.
+
+*Note:* Running a `docker` command will cause the docker daemon to be started if it was not already started.
+
+== Taking down the Virtual Machine
+
+Let's now get rid of that VM so we can start again from scratch.
+First escape out of the serial console by pressing `CTRL + ]` and then type:
+
+[source,bash]
+----
+[host]$ virsh destroy fcos
+[host]$ virsh undefine --remove-all-storage fcos
+----
+
+== Booting Fedora CoreOS: Intermediate Provisioning Scenario
```

Comment:
I think that the content from here on already belongs to the other tutorial page.

Reply:
https://github.com/coreos/fedora-coreos-docs/pull/134/commits/3b9e4bade5156345df4ffa1f79270fd723d8f1d1 Fixed